<launch>
  <node name="cloud_tansformer"
        pkg="pclTransform" type="cloud_transformer" output="screen">
    <remap from="/inputPC" to="/camera/depth_registered/points" />
    <remap from="/outputPC" to="/croppedPointCloud" />
    <param name="frameIn" type="str" value="camera_depth_frame" />
    <param name="frameOut" type="str" value="virtual_base_link" />
  </node> 
  <node name="dummy_handeye" pkg="tf" type="static_transform_publisher"
            args="-0.15122919502 -1.33268123562 0.382354381499 -0.138836537909 0.0784394799313 0.691963622156 0.704100851685 wrist_3_link camera_link 10" />
</launch>
  <node name="virtual_base" pkg="tf" type="static_transform_publisher"
            args="0.0 0.0 0.0 0.0 1.0 0.0 0.0 base_link virtual_base_link 10" />
</launch>



